histolab/filters/image_filters.py

```diff
@@ -99,7 +98,9 @@ def __init__(self, lambd: Callable[[PIL.Image.Image], PIL.Image.Image]) -> None:
         assert callable(lambd), repr(type(lambd).__name__) + " object is not callable"
         self.lambd = lambd
 
-    def __call__(self, img: PIL.Image.Image) -> Union[PIL.Image.Image, np.ndarray]:
+    def __call__(self, img: ImageOrNPArray) -> ImageOrNPArray:
+        assert isinstance(img, PIL.Image.Image), "Input must be an image"
```

Comment:
best practice: we don't want to have assertion statements in code but only in the tests, better to raise an appropriate exception

Reply:
oh the assert here is just for making mypy happy :D I think it will go away with some refactoring 😊